packages/vite/src/index.ts

```diff
@@ -18,7 +18,7 @@ import { getConfig, getPaths } from '@redwoodjs/project-config'
 const readFile = promisify(fsReadFile)
 
 // Using require, because plugin has TS errors
-const commonjs = require('vite-plugin-commonjs')
+const { default: commonjs } = require('vite-plugin-commonjs')
```

Comment:
Changing this to an import statement might be the solution, they might have fixed the TS problems here now (or we can add ts-ignore)